app/main.py

```diff
@@ -1,13 +1,12 @@
-# from models import Actor
-# from managers import ActorManager
-#
-# if __name__ == "__main__":
-#     Actor.objects = ActorManager()
-#
-#     Actor.objects.create(first_name="Emma", last_name="Watson")
-#     Actor.objects.create(first_name="Daniel", last_name="Radclife")
-#     print(Actor.objects.all())
-#     Actor.objects.update(2, "Daniel", "Radcliffe")
-#     print(Actor.objects.all())
-#     Actor.objects.delete(1)
-#     print(Actor.objects.all())
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
+    Actor.objects.create(first_name="Emma", last_name="Watson")
```

Comment:
I think your table will not be created (an error will occur) because you are using parameters named `new_first_name` and `new_last_name` in the `managers.py` module, and there are no such parameters in this module